experimental/content-type-screen-effect/README.md

```diff
@@ -0,0 +1,92 @@
+# Screen effect content type
+
+This experimental package provides a content type to support adding screen effects to messages.
+
+## What’s a screen effect?
+
+A screen effect is something that happens on a particular trigger, such as sending balloons along with a happy birthday message or sending snowflakes to accompany a holiday message.
+
+## Why screen effects?
+
+Providing screen effects can be a way to surprise and delight an app's users, by presenting them with an unexpected visual effect.
+
+## Install the package
+
+```bash
+# npm
+npm i @xmtp/experimental-content-type-screen-effect
+
+# yarn
+yarn add @xmtp/experimental-content-type-screen-effect
+
+# pnpm
+pnpm i @xmtp/experimental-content-type-screen-effect
+```
+
+## Create a screen effect
+
+Screen effects are represented as objects with the following keys:
+
+- `messageId`: The message id for the message that the screen effect is being sent with
+- `effectType`: The type of effect (currently `SNOW` or `RAIN`)
```

Comment:
Maybe we can add a comment that eludes to opening a PR if you want more effects?

```suggestion
- `effectType`: The type of effect (currently `SNOW` or `RAIN` open a PR if you'd like more effects)
```

Reply:
i like that idea — updated!